config/src/config/indexer_table_info_config.rs

```diff
@@ -20,6 +21,12 @@ pub struct IndexerTableInfoConfig {
     pub parser_batch_size: u16,
 
     pub enable_expensive_logging: bool,
+
+    /// Enable backup service
+    pub db_backup_enabled: bool,
+
+    /// Backup and restore service config
+    pub gcs_bucket_name: String,
```

Comment:
should we make this optional and use None as the default value here, to avoid accidentally store data from different environment to the same bucket?

Reply:
done; add explicit checking if chain id doesn't match.

Reply:
done.